msm-sitemap.php

```diff
@@ -803,7 +865,7 @@ public static function get_supported_post_types() {
 		return apply_filters( 'msm_sitemap_entry_post_type', array( 'post' ) );
 	}
 
-	private static function get_supported_post_types_in() {
+	public static function get_supported_post_types_in() {
```

Comment:
We have replicated the query construction as in date_range_has_posts, so that we are maintaining a similar query and as a result are using get_supported_post_types_in to construct that.